scripts/do.sh

```diff
@@ -16,20 +16,11 @@ https://github.com/getsentry/devenv#install
     exit 1
 fi
 
-if [[ ${GETSENTRY:-0} -eq 1 ]]; then
```

Comment:
soooo this is still needed, but not if i move make setup-git into `devenv sync`: https://github.com/getsentry/getsentry/pull/13540/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R28